pkg/logcli/query/query.go

```diff
@@ -161,6 +161,8 @@ func (q *Query) printResult(value loghttp.ResultValue, out output.LogOutput, las
 		q.printScalar(value.(loghttp.Scalar))
 	case loghttp.ResultTypeMatrix:
 		q.printMatrix(value.(loghttp.Matrix))
+	case loghttp.ResultTypeHistogramMatrix:
```

Comment:
using the labels we should be able to not have to use a new response model WDYT ?

Reply:
@cyriltovena Have removed custom response format. Output will be of the form

```json
[
  {
    "metric": {
      "path": "/api/v2/"
      "le": "0.05",
    },
    "values": [
      [
        1653907528, 
        "2"
      ],
      [
        1653907542, 
        "3"
      ]
    ]
  },
  {
    "metric": {
      "path": "/api/v2/"
      "le": "0.025",
    },
    "values": [
      [
        1653907528, 
        "2"
      ],
      [
        1653907542, 
        "3"
      ]
    ]
  }
]
```